### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by Prakharsvnit**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -10,3 +10,6 @@ phonenumbers==7.2.6
 django-cities-light==3.2.0
 Unidecode==0.4.19
 django-braces==1.9.0
+mccabe==0.6.1,
+pycodestyle==2.3.1,
```
Comment (tapasweni-pathak) [inline requirements.txt]: say flake8 and specify its version here. Also include both #480 and #481 in a single pull request. In .travis.yml run flake8 on vms.

**Comment by Prakharsvnit** [comment]
I am having problem with travis [build](https://travis-ci.org/Prakharsvnit/vms/builds/288258357) with error:
**The command "coverage run --source='.' manage.py test" exited with 1** .

File: .travis.yml [hunk 2]
```diff
@@ -12,8 +12,10 @@ before_script:
   - sleep 3
 script:
   - cd vms
+  - flake8 cmdbox
   - python manage.py syncdb --noinput
   - python manage.py migrate --noinput --traceback --settings=vms.settings
   - coverage run --source='.' manage.py test
+  - coverage run manage.py test --settings=vms.settings
```
Comment (tapasweni-pathak) [inline .travis.yml]: Remove this one here.

**Comment by Prakharsvnit** [comment]
I am still facing the same [error](https://travis-ci.org/Prakharsvnit/vms/builds/288296710) even after removing the line you suggested

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak I am getting the same error. It is because in requirements.txt file we have installed selenium web driver version 2.53.1 but the latest mozilla firefox version has some compatibilty issues with it. But the problem can't be solved just by changing the version of selenium driver version because those versions require geckodriver to be downloaded and installed too.

**Comment by anjali-dhanuka** [comment]
ikj

**Pull request closed without merging**